bin/sauce-pre-upload.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# check if CIRCLE_BRANCH variable is set
+if [[ -n "$CIRCLE_BRANCH" ]] ; then
+    # replace / with - and assign to new var SAUCE_FILENAME
+    export SAUCE_FILENAME=${CIRCLE_BRANCH//[\/]/-};
+ else
+    echo "Expected CIRCLE_BRANCH env variable";
```

Comment:
Would it be useful/approrpiate to have something like `exit 1` here?

Reply:
🤔 Hmm I guess the added value would be other scripts could then use the exit code as a check, I don't feel too strongly about it so I can go ahead and make that update if you think it's best to make that distinction.

Reply:
I didn't know if exiting immediately here might make the cause of the failure more obvious. But I'm not familiar enough with how this script is used to know if that change would actually help, so if you don't think it would be helpful, I wouldn't bother making the change.

Reply:
It's only used here https://github.com/wordpress-mobile/gutenberg-mobile/pull/1139/files/579773229a76b2ab75999b04669fc9ff92d5f743#diff-1d37e48f9ceff6d8030570cd36286a61R141 locally it's expected you can prefix the command with the env var if you're triggering a CI run

Reply:
👍  I'm fine if you want to leave this as-is.